packages/react/src/components/Avatar/Avatar.stories.mdx

```diff
@@ -17,6 +19,8 @@ export const Template = args => <Avatar {...args} />;
 - [Overview](#overview)
 - [Props](#props)
 - [Usage](#usage)
+- [Variants](#variants)
+  - [Random Colors](#random-colors)
```

Comment:
Is the tab intentional ?

Reply:
Yes. `Random Colors` is under `Variants`.